docs/source/conf.py

```diff
@@ -2621,6 +2632,15 @@
     "RemoteException",
     # torch.distributed.rpc.rref_proxy
     "RRefProxy",
+    # torch.distributed.tensor.api
+    "DTensor",
```

Comment:
If you have to add things here, it means it's not showing on the website.
You should not add anything to this file ever in general as it means there is some public API that is not documented being added.

If you properly updated the .rst to show your docstring, nothing is needed here.